src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/MessageHandlerConfig.java

```diff
@@ -53,36 +49,18 @@
 
     @PostConstruct()
     public void registerMessageHandlers() throws InterruptedException, ServiceBusException {
-        try {
-            if (readNotificationsQueueClient != null) {
-                readNotificationsQueueClient.registerMessageHandler(
-                    errorNotificationHandler,
-                    messageHandlerOptions,
-                    notificationsReadExecutor
-                );
-            }
-
-            processedEnvelopesQueueClient.registerMessageHandler(
-                processedEnvelopeNotificationHandler,
+        if (readNotificationsQueueClient != null) {
+            readNotificationsQueueClient.registerMessageHandler(
+                errorNotificationHandler,
                 messageHandlerOptions,
-                processedEnvelopesReadExecutor
+                notificationsReadExecutor
             );
-        } catch (InterruptedException e) {
-            Thread.currentThread().interrupt();
-            handleMessageHandlerRegistrationError(e);
-        } catch (ServiceBusException e) {
-            handleMessageHandlerRegistrationError(e);
         }
-    }
 
-    private <T extends Exception> void handleMessageHandlerRegistrationError(T cause) throws T {
-        if (failOnMessageHandlerRegistrationError) {
-            throw cause;
-        } else {
-            // The application has to keep working on Preview - otherwise the pipeline
-            // wouldn't create queues which it relies on.
-            // The problem will be addresses in BPS-445
-            log.error("An error occurred when trying to register message handlers", cause);
-        }
+        processedEnvelopesQueueClient.registerMessageHandler(
+            processedEnvelopeNotificationHandler,
+            messageHandlerOptions,
+            processedEnvelopesReadExecutor
+        );
```

Comment:
should this block be treated same as the one above? (if condition wrap)

Reply:
The condition is about reading from `notifications` queue. I thought it had some business meaning, e.g. we may want to turn it off or sth. If that's for the purpose of making the behaviour environment-specific, I'd say we should remove it and always register the handler.